src/plfa/part2/Confluence.lagda.md

```diff
@@ -401,105 +401,108 @@ sub-par : ∀{Γ A B} {N N′ : Γ , A ⊢ B} {M M′ : Γ ⊢ A}
 sub-par pn pm = subst-par (par-subst-zero pm) pn
 ```
 
-
 ## Parallel reduction satisfies the diamond property
 
 The heart of the confluence proof is made of stone, or rather, of
 diamond!  We show that parallel reduction satisfies the diamond
 property: that if `M ⇛ N` and `M ⇛ N′`, then `N ⇛ L` and `N′ ⇛ L` for
-some `L`.  The proof is relatively easy; it is parallel reduction's
-_raison d'etre_.
+some `L`.  The typical proof is an induction on `M ⇛ N` and `M ⇛ N′`
+so that every possible pair gives rise to a witeness `L` given by
+performing enough beta reductions in parallel.
+
+Will it be even nicer if we simply perform as many beta reductions in
+parallel as possible on `M`, say `M ⁺`, so that `N ⇛ M ⁺` holds
+independent of another `N′`? This is exactly the idea of Takahashi's
+complete development.
```

Comment:
The sentence structure is a bit surprising. How about the following instead:

However, a simpler approach  is to perform as many beta reductions in parallel as
possible on `M`, say `M ⁺`, and then show that `N` also parallel reduces to `M ⁺`.
This is the idea of Takahashi's *complete development*.

Reply:
Done. Thanks for rephrasing it!